src/lib/block/View/RecordList.vue

```diff
@@ -1,9 +1,32 @@
 <template>
   <div v-if="recordListModule">
     <span v-if="!options.hideAddButton && recordListModule.canCreateRecord">
-      <router-link
-                  class="btn-url d-inline-block mt-1"
-                  :to="{ name: 'page.record.create', params: { pageID: options.pageID, refRecord: record }, query: null }">+ {{ $t('block.recordList.addRecord') }}</router-link>
+      <router-link class="float-left"
+                   :to="{
+                     name: 'page.record.create',
+                     params: { pageID: options.pageID, refRecord: record },
+                     query: null,
+                    }">
+
+        <b-button variant="outline-primary"
```

Comment:
button inside router-link? and why do both have float-left class?

it's probably best to add float-left class simply to span instead of each button

Reply:
Each button has a float left, because import/export components contain both the button and the modal, so they have to be wrapped in a div